### DISCUSSION TITLE
Next.js vs Svelte in mobile development

### DISCUSSION THREAD

**Discussion by GKosheev** [tauri-apps/tauri#6351] [Q&A] [answered]
I'm new to Tauri and I don't know deeply about how it completely works. I'm using Tauri for mobile development. Since mobile and web are quite different, I wonder if all the differences between Next.js and Svelte, including the performance, on the web are the same in mobile.

So in other words, if I compile the Svelte project to an Android app, would it have the same performance advantages over Next.js as in the web?

**Comment by FabianLars** [comment, marked as answer]
> So in other words, if I compile the Svelte project to an Android app, would it have the same performance advantages over Next.js as in the web?

Probably yes. tauri on mobile is quite similar to tauri on desktop, as in it uses webviews (slimmed down browsers) for your app, so the performance differences between frameworks should be similar-ish.
